Source/bmalloc/libpas/src/libpas/pas_debug_heap.h

```diff
@@ -38,12 +38,14 @@ PAS_BEGIN_EXTERN_C;
 
 #if PAS_BMALLOC
 
+#include "BExport.h"
+
 /* The implementations are provided by bmalloc. */
-PAS_API extern bool pas_debug_heap_is_enabled(pas_heap_config_kind kind);
-PAS_API extern void* pas_debug_heap_malloc(size_t size);
-PAS_API extern void* pas_debug_heap_memalign(size_t alignment, size_t size);
-PAS_API extern void* pas_debug_heap_realloc(void* ptr, size_t size);
-PAS_API extern void pas_debug_heap_free(void* ptr);
+BEXPORT extern bool pas_debug_heap_is_enabled(pas_heap_config_kind);
+BEXPORT extern void* pas_debug_heap_malloc(size_t);
+BEXPORT extern void* pas_debug_heap_memalign(size_t alignment, size_t);
+BEXPORT extern void* pas_debug_heap_realloc(void* ptr, size_t);
+BEXPORT extern void pas_debug_heap_free(void* ptr);
```

Comment:
The separation between `PAS` and `B` seems generally confusing. Not sure if there's anything actionable or blocking there.